public/components/common/form/index.tsx

```diff
@@ -7,13 +7,12 @@ import { InputFormSwitch } from './input_switch';
 import { InputFormFilePicker } from './input_filepicker';
 import { InputFormTextArea } from './input_text_area';
 import { EuiFlexGroup, EuiFlexItem, EuiFormRow } from '@elastic/eui';
-import { OsCard } from '../../../controllers/register-agent/components/os-card/os-card';
-import { SettingTypes } from './types';
 
-interface InputFormProps {
-  type: SettingTypes;
+export interface InputFormProps {
+  type: string;
   value: any;
   onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onOptionChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
**nitpick**: it is not required. We could remove it.

Reply:
Remove in commit [61aa4b3](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/61aa4b38189c61c1af0c353d8fca9c6a7a37fb18)